src/LaunchDarkly.ServerSdk.Redis/LaunchDarkly.ServerSdk.Redis.csproj

```diff
@@ -1,20 +1,20 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <Version>1.1.1</Version>
-    <TargetFrameworks>netstandard1.6;netstandard2.0;net45</TargetFrameworks>
+    <Version>2.0.0</Version>
+    <TargetFrameworks>net461;net472;net48;netcoreapp2.0;netcoreapp2.1;netcoreapp3.0;netcoreapp3.1;netstandard2.1;netstandard2.0;</TargetFrameworks>

```

Comment:
What's the rationale for defining all these extra target frameworks? If we provide just `net461`, that ought to work fine in apps that use `net472`, just as we were compatible with .NET Framework 4.5 _and above_ when we had `net45`. Similarly, `netstandard2.0` should be adequate to support all of those `netcoreapp` versions.

Reply:
Yep, you're right.

Reply:
It's possible that StackExchange.Redis has implemented something differently in, for instance, their `net472` target version that makes it work better in .NET Framework 4.7.2 than their `net46` version would. But, as I understand it, that shouldn't require us to have a corresponding version; an application that targets `net472` would end up getting the `net461` version of `LaunchDarkly.ServerSdk.Redis`, and the `net472` version of `StackExchange.Redis`.